### PR TITLE
Extract attribute helper from buyer frontend

### PR DESCRIPTION
**Pull request by quis**
The idea is that all frontend apps will use this to transform data from the API before sending it to the toolkit templates.

### CODE REVIEW

File: dmutils/service_attribute.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+class Attribute(object):
+    """Wrapper to handle accessing an attribute in service_data"""
+
+    def __init__(
+        self,
+        value,
+        question_type,
+        label='',
+        optional=False,
+    ):
```
Comment (robyoung) [inline dmutils/service_attribute.py]: ):

File: tests/test_service_attribute.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+)
+
+
+class TestAttribute(unittest.TestCase):
+
+    def test_Attribute_works_if_the_key_is_in_service_data(self):
+        try:
+            attribute = Attribute('supportAvailability', 'text')
+        except KeyError:
+            self.fail("'supportAvailability' key should be in service data")
```
Comment (robyoung) [inline tests/test_service_attribute.py]: I'm struggling to see what this test tells us.

Comment (quis) [inline tests/test_service_attribute.py]: @robyoung Good point. It's a hangover. Removed in 4bd9829.

**Comment by robyoung** [comment]
:+1:

**Pull request merged**